docs/concepts/activation.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Activating environments
+description: How an environment is activated, and how to make the most of it
+---
+
+# Activating environments
+
+[Environments][environment-concept] are a central concept in Flox,
+representing the tools you want to use, all of their dependencies,
+various environment variables necessary to make them function properly,
+_and_ all of your customizations.
+Given that environments are such an important part of Flox,
+it stands to reason that _how you use them_ is also an important part of Flox.
+
+There are three different ways to use an environment,
+and two different modes that an environment can be activated in.
+At the end of the day, though, it all boils down to properly configuring a
+shell.
+The `hook` and `profile` scripts specified in your manifest are run as part of
+configuring that shell.
+Understanding when and why they're run will help you take full advantage
+of Flox,
+so let's walk through what it means to "activate" an environment and how it
+works.
+
+## Configuring the shell
+
+When you "activate" an environment,
+Flox configures a shell,
+making all of the packages and environment variables specified in your
+manifest available, as mentioned above.
+
+The most basic way to activate an environment is simply by calling
+`flox activate`,
+which puts you into a subshell with everything configured:
+
+```bash
+$ flox activate
+flox [myenv] $ # Now I can use my packages
+```
+
+One of the core features that makes Flox so attractive for development is that
+the packages in your manifest are available when the environment is active,
+and they're gone when it's inactive.
+We do this by carefully setting a collection of environment variables,
+some of which you may be familiar with, such as `PATH`, and others which you
+may not have heard of like `ACLOCAL_PATH`, `RUST_SRC_PATH`, and others.
+
+As an example, if you create an environment `myenv` and install `hello`,
+Flox will place `<path to myenv>/.flox/run/<your system>.myenv.dev/bin` at the
+beginning of your `PATH` variable so that `hello` will be selected from your
+environment rather than from elsewhere on your system.
+
+## Three different ways to activate
+
+I mentioned above that there are three different ways to use an environment.
```

Comment:
nit: I think 1st person sounds a little out of place here since it's docs not a blog